src/test/functional/steps/opening-hours.ts

```diff
@@ -87,6 +87,19 @@ When('I enter an incomplete opening hour description', async () => {
   await I.setElementValueAtIndex('#openingTimesTab input[name$="[hours]"', entryFormIdx, '10:00am to 4:00pm', 'input');
 });
 
+When('I left the opening hours blank and select description at index {int}', async (typeId: number) => {
+  const numFieldsets = await I.countElement('#openingTimesTab fieldset');
+  const entryFormIdx = numFieldsets - 2; // we deduct one each for zero-based index and hidden template field
+
+  const typeSelector = '#openingTimesTab select[name$="[type_id]"]';
+  const hoursSelector = '#openingTimesTab input[name$="[hours]"]';
+
+  await I.setElementValueAtIndex(typeSelector, entryFormIdx, type_id, 'select');
```

Comment:
Think this change was needed too.
```suggestion
  const typeSelector = '#openingTimesTab select[name$="[typeId]"]';
  const hoursSelector = '#openingTimesTab input[name$="[hours]"]';

  await I.setElementValueAtIndex(typeSelector, entryFormIdx, typeId, 'select');
```

Reply:
line 97 change is fine but [name$="[type_id]"] from line 94 is the part of the name of the element